pkgs/applications/version-management/git/default.nix

```diff
@@ -1,48 +1,91 @@
-{ fetchurl, fetchpatch, lib, stdenv, buildPackages
-, curl, openssl, zlib, expat, perlPackages, python3, gettext, cpio
-, gnugrep, gnused, gawk, coreutils # needed at runtime by git-filter-branch etc
-, openssh, pcre2, bash
-, asciidoc, texinfo, xmlto, docbook2x, docbook_xsl, docbook_xml_dtd_45
-, libxslt, tcl, tk, makeWrapper, libiconv, libiconvReal
-, svnSupport ? false, subversionClient, perlLibs, smtpPerlLibs
-, perlSupport ? stdenv.buildPlatform == stdenv.hostPlatform
-, nlsSupport ? true
-, osxkeychainSupport ? stdenv.hostPlatform.isDarwin
-, guiSupport ? false
-, withManual ? true
-, pythonSupport ? true
-, withpcre2 ? true
-, sendEmailSupport ? perlSupport
-, Security, CoreServices
-, nixosTests
-, withLibsecret ? false
-, pkg-config, glib, libsecret
-, gzip # needed at runtime by gitweb.cgi
-, withSsh ? false
-, sysctl
-, doInstallCheck ? !stdenv.hostPlatform.isDarwin  # extremely slow on darwin
-, tests
+{
+  asciidoc,
+  bash,
+  buildPackages,
+  CoreServices,
+  coreutils, # needed at runtime by git-filter-branch etc
+  cpio,
+  curl,
+  docbook2x,
+  docbook_xml_dtd_45,
+  docbook_xsl,
+  doInstallCheck ? !stdenv.hostPlatform.isDarwin, # extremely slow on darwin
+  expat,
+  fetchpatch,
+  fetchurl,
+  gawk,
+  gettext,
+  glib,
+  gnugrep,
+  gnused,
+  guiSupport ? false,
+  gzip, # needed at runtime by gitweb.cgi
+  lib,
+  libiconv,
+  libiconvReal,
+  libsecret,
+  libxslt,
+  makeWrapper,
+  nixosTests,
+  nlsSupport ? true,
+  openssh,
+  openssl,
+  osxkeychainSupport ? stdenv.hostPlatform.isDarwin,
+  pcre2,
+  perlLibs,
+  perlPackages,
+  perlSupport ? stdenv.buildPlatform == stdenv.hostPlatform,
+  pkg-config,
+  python3,
+  pythonSupport ? true,
+  Security,
+  sendEmailSupport ? perlSupport,
+  smtpPerlLibs,
+  stdenv,
+  subversionClient,
+  svnSupport ? false,
+  sysctl,
+  tcl,
+  tests,
+  texinfo,
+  tk,
+  withLibsecret ? false,
+  withManual ? true,
+  withpcre2 ? true,
+  withSsh ? false,
+  xmlto,
+  zlib,
 }:
 
 assert osxkeychainSupport -> stdenv.hostPlatform.isDarwin;
 assert sendEmailSupport -> perlSupport;
 assert svnSupport -> perlSupport;
 
 let
-  version = "2.46.0";
+  version = "2.47.0";
```

Comment:
The version at `staging` has moved to 2.46.2:
- https://github.com/NixOS/nixpkgs/pull/344393